examples/Cargo.toml

```diff
@@ -43,9 +43,10 @@ env_logger = "0.7"
 glsl-to-spirv = "0.1.4"
 
 [target.'cfg(target_arch = "wasm32")'.dependencies]
-wasm-bindgen = "0.2.55"
+wasm-bindgen = "0.2"
 console_error_panic_hook = "0.1.6"
 console_log = "0.1.2"
+stdweb = "0.4"
```

Comment:
isn't it an alternative to wasm-bindgen? why do we need to depend on both?

Reply:
True, not needed. It was added for some tests.